src/classes/GE_GiftEntryController.cls

```diff
@@ -52,6 +69,24 @@ public with sharing class GE_GiftEntryController {
         set;
     }
 
+    @AuraEnabled
+    public static GiftBatchView getGiftBatchView(Id giftBatchId){
+        try {
+            return giftBatchService.viewModelFrom(giftBatchId);
+        } catch (AuraHandledException e) {
+            throw new AuraHandledException(e.getMessage());
+        }
+    }
+
+    @AuraEnabled
+    public static GiftView getGiftView(Id giftId){
```

Comment:
It feels a bit strange to call the param "giftId" when the Id, in truth, belongs to a DataImport? But not sure if that feeling stands up to further questioning.

Reply:
We could use a value object called "GiftBatchId"

Reply:
I'll update these to use GiftId and GiftBatchId value objects!